packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -568,23 +569,23 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		return response;
 	}
 
-	private async onUploadReject(localId: string, error: any) {
+	private async onUploadReject(localId: string, error: any, count: number) {
```

Comment:
can we simplify this, maybe use driver-utils runWithRetry instead?

Reply:
maybe something like this:
```
	private async uploadBlob(
		localId: string,
		blob: ArrayBufferLike,
	): Promise<ICreateBlobResponse | void> {
		return runWithRetry(
			async () => this.getStorage().createBlob(blob),
			"createBlob",
			this.mc.logger,
			{
				cancel: this.pendingBlobs.get(localId)?.abortSignal,
			},
		).then((response) => this.onUploadResolve(localId, response));
	}
```

Reply:
I like using runWithRetry method. There is some cleanup to do in case of ending up rejecting but I can add that to the method.